### PR TITLE
MGMT-3865 Block VIP DHCP allocation with IPv6 machine network

### PR DESCRIPTION
**Pull request by empovit**
This combination is currently not supported

### CODE REVIEW

File: internal/bminventory/inventory_test.go [hunk 1]
```diff
@@ -4737,6 +4796,13 @@ func verifyApiError(responder middleware.Responder, expectedHttpStatus int32) {
 	ExpectWithOffset(1, conncreteError.StatusCode()).To(Equal(expectedHttpStatus))
 }
 
+func verifyApiErrorString(responder middleware.Responder, expectedHttpStatus int32, expectedSubstring string) {
+	ExpectWithOffset(1, responder).To(BeAssignableToTypeOf(common.NewApiError(expectedHttpStatus, nil)))
+	conncreteError := responder.(*common.ApiErrorResponse)
```
Comment (osherdp) [inline internal/bminventory/inventory_test.go:4801]: typo?

File: internal/cluster/validations/validations.go [hunk 2]
```diff
@@ -411,3 +412,18 @@ func getRegistriesWithAuth(ignoreList string, ignoreSeparator string, images ...
 
 	return &registries, nil
 }
+
+//ValidateVipDHCPAllocationWithIPv6 returns an error in case of VIP DHCP allocation
+//being used with IPv6 machine network
+func ValidateVipDHCPAllocationWithIPv6(vipDhcpAllocation bool, machineNetworkCIDR string) error {
+
```
Comment (osherdp) [inline internal/cluster/validations/validations.go:419]: delete blank line

**Review by osherdp — CHANGES_REQUESTED** [review]

File: internal/bminventory/inventory_test.go (hunk 1 above)
Comment (empovit) [reply internal/bminventory/inventory_test.go:4801]: Right, a copied one :) Fixing

Comment (empovit) [reply internal/bminventory/inventory_test.go:4801]: Strange VSC spellchecker has nothing against the "alternative" spelling

**Comment by osherdp** [comment]
@empovit maybe it's not a flake?
It seems like a relevant test also

**Comment by empovit** [comment]
> @empovit maybe it's not a flake?
> It seems like a relevant test also

Thanks, will look into it

**Pull request merged**